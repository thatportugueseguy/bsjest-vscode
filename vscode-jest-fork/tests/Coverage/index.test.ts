jest.disableAutomock()

describe('Coverage', () => {
  describe('module.exports', () => {
    it('should match the snapshot', () => {
      expect(require('../../src/Coverage')).toMatchSnapshot()
    })
  })
})
