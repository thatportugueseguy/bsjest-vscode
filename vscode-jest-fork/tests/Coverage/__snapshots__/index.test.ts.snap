// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coverage module.exports should match the snapshot 1`] = `
Object {
  "CoverageMapProvider": [Function],
  "CoverageOverlay": [Function],
  "registerCoverageCodeLens": [Function],
}
`;
